AntWars/ConfigurationPanel.cs

```diff
@@ -584,11 +586,42 @@ private void btn_player1loadAI_Click(object sender, EventArgs e)
                 lbl_player1AIPath.Text = res;
                 configLoader.get().Player1.AIPath = res;
             }
+            checkPlayerKI();
         }
 
         private void numeric_gameConfigMaxTicks_ValueChanged(object sender, EventArgs e)
         {
             configLoader.get().Game.MaxTicks = Convert.ToInt32(numeric_gameConfigMaxTicks.Value);
         }
+
+        private void checkPlayerKI()
+        {
+            // TODO1: Player.AIPath muss nach übernahme der Properties von Player1/2 zur AI aus Gameconfig bezogen werden
+            // TODO2: wenn TODO1 erfüllt try catch entfernen
+            try
+            {
+                if (!String.IsNullOrEmpty(configLoader.get().Player1.AIPath))
+                {
+                    btn_player1loadAI.BackColor = Color.LightGreen;
+                }
+                else
+                {
+                    btn_player1loadAI.BackColor = Color.Red;
+                }
+
+                if (!String.IsNullOrEmpty(configLoader.get().Player2.AIPath))
+                {
+                    btn_player2loadAI.BackColor = Color.LightGreen;
+                }
+                else
+                {
+                    btn_player2loadAI.BackColor = Color.Red;
+                }
+            }
+            catch (System.NullReferenceException)
```

Comment:
Ist meiner Meinung nach nicht nötig, da durch die Änderungen von @nicolasries , dass die PlayerAIPaths in die Gameconfig kommen, ich keine Exceptions mehr bekommen kann, da durch Aufruf des "Neu"-Buttons die Gameconfig initializiert wird.